Source/Urho3D/Graphics/OpenGL/OGLGraphics.cpp

```diff
@@ -2682,20 +2720,27 @@ unsigned Graphics::GetFloat32Format()
 
 unsigned Graphics::GetLinearDepthFormat()
 {
-#ifndef GL_ES_VERSION_2_0
+#ifndef GL_ES_VERSION_2_0 
     // OpenGL 3 can use different color attachment formats
     if (gl3Support)
         return GL_R32F;
+    else
 #endif
+#ifdef GL_ES_VERSION_3_0
+    return GL_R16F;

```

Comment:
This statement has wrong indentation in relation to previous `else`. Seems to me that removing `else` would be enough to maintain same behavior while not having confusing indentation.